### DISCUSSION TITLE
Sort by JSON fields

### DISCUSSION THREAD

**Discussion by catalinux17** [pocketbase/pocketbase#645] [Q&A] [answered]
Is there a way to make a query to sort inner JSON of an collection object?

Let's say we have this collection:
`
shop: String
product: JSON
`

If I added the following object:
`
shop: "Intersport",
product:
{
 "brand": "adidas",
 "price": 200
}
`

Is it possible to do a query like product.brand ~ "adidas"?

**Comment by ganigeorgiev** [comment, marked as answer]
Unfortunately no at the moment.

Filter and sort capabilities to the `json` type fields will be added with https://github.com/pocketbase/pocketbase/issues/423.

**Reply by catalinux17** [reply]
Thanks!!
